tests/unit/ValidationUtilsTest.js

```diff
@@ -313,4 +313,17 @@ describe('ValidationUtils', () => {
             expect(ValidationUtils.isValidAccountRoute('123aaa')).toBe(false);
         });
     });
+
+    describe('ValidatePersonName', () => {
+        test('Valid person name', () => {
+            expect(ValidationUtils.isValidPersonName('test name')).toBe(true);
+            expect(ValidationUtils.isValidPersonName(`X Æ A test`)).toBe(true);
+        });
```

Comment:
Can we add a quick test for a hyphenated last name such as `a hypenated-name`? This is fairly common so I just want to make sure it doesn't ever get broken